store/modules/stimulation/actions.js

```diff
@@ -34,7 +34,7 @@ export default {
           y: [setting.phase_one_charge],
         };
         // If biphasic, handle remaining pulse components
-        if (setting.interphase_interval) {
+        if (setting.phase_two_duration || setting.phase_two_duration == 0) {
```

Comment:
I think you can just check `setting.phase_two_duration != null` https://stackoverflow.com/questions/2647867/how-can-i-determine-if-a-variable-is-undefined-or-null